EosGround/config/eos_schema.sql

```diff
@@ -0,0 +1,96 @@
+-- SCHEMA: eos_schema
+
+-- DROP SCHEMA IF EXISTS eos_schema ;
+
+CREATE SCHEMA IF NOT EXISTS eos_schema
+    AUTHORIZATION pg_database_owner;
+
+GRANT USAGE ON SCHEMA eos_schema TO PUBLIC;
+
+GRANT ALL ON SCHEMA eos_schema TO pg_database_owner;
+
+-- This script was generated by the ERD tool in pgAdmin 4.
+-- Please log an issue at https://redmine.postgresql.org/projects/pgadmin4/issues/new if you find any bugs, including reproduction steps.
+BEGIN;
+
+
+CREATE TABLE IF NOT EXISTS eos_schema.received_data
+(
+    id bigint NOT NULL GENERATED ALWAYS AS IDENTITY,
+    raw_bytes bytea NOT NULL,
+    rssi smallint NOT NULL,
+    processed boolean NOT NULL,
+    CONSTRAINT received_data_pkey PRIMARY KEY (id)
+);
+
+CREATE TABLE IF NOT EXISTS eos_schema.received_packets
+(
+    id bigint NOT NULL GENERATED ALWAYS AS IDENTITY,
+    data_id bigint NOT NULL,
+    type smallint NOT NULL,
+    sender smallint NOT NULL,
+    priority smallint NOT NULL,
+    destination smallint NOT NULL,
+    generate_time timestamp without time zone NOT NULL,
+    sequence_number smallint NOT NULL,
+    send_time timestamp without time zone NOT NULL,
+    received_time timestamp without time zone NOT NULL,
+    packet_body bytea NOT NULL,
+    processed boolean NOT NULL,
+    CONSTRAINT received_packets_pkey PRIMARY KEY (id)
+);
+
+CREATE TABLE IF NOT EXISTS eos_schema.temperature
+(
+    id bigint NOT NULL GENERATED ALWAYS AS IDENTITY,
+    packet_id bigserial NOT NULL,
```

Comment:
bigserial is auto-incremented, do not use it for foreign keys
this applies to some other tables in this script as well

Reply:
Fixed